regulations/static/regulations/css/scss/partials/_navigation.scss

```diff
@@ -37,13 +37,12 @@ Appendix Navigation
 
 .appendix-nav a:link,
 .appendix-nav a:visited {
-  color: #101820;
+  color: $black;
```

Comment:
Is `$black` the main text color? I think the intent here was to match the main text.